README.md

```diff
@@ -120,13 +120,12 @@ Then point your browser to http://localhost:3000/.
 
 ## Local Testing
 
-To run a build that can be locally tested using any HTTP server, run
+To run a build that can be locally tested using any HTTP server:
 
-```
-npm run build:local
-```
-
-Then serve the top-level repository directory, and browse to http://localhost:<port>/material-components-web-catalog/.
+1. `npm run build`
+2. Rename the `build` folder to `material-components-web-catalog`
```

Comment:
I know that's necessary but seeing it written out feels so weird. I wish there was a different way.

Reply:
I previously had a separate `build:local` task and one of the things it did was output to this folder instead automatically, but that was since removed during the switch to hash routing. I dunno if it's worth putting it back for this one thing.

Reply:
Probably not, especially if we don't use it.